src/library/libraryquery.cpp

```diff
@@ -181,6 +165,33 @@ LibraryQuery::LibraryQuery(const QueryOptions& options)
   }
 }
 
+int LibraryQuery::GetSecondsFromToken(QString val) {
```

Comment:
`int LibraryQuery::GetSecondsFromToken(const QString& val) const`

Reply:
I am not well versed in C++, so I am unsure about the semantics, and if this is also needed in the corresponding .h file, but I have added it there as well.